spec/support/capybara.rb

```diff
@@ -44,3 +44,4 @@
 
 Capybara.javascript_driver = :chrome
 Capybara.default_max_wait_time = CAPYBARA_TIMEOUT
+Capybara.raise_server_errors = false
```

Comment:
Do we really need this @gutakk

Reply:
when I tried to test `show` template which has google html will raise following error
```ruby
Failure/Error: raise ActionController::RoutingError, "No route matches [#{env['REQUEST_METHOD']}] #{env['PATH_INFO'].inspect}"
     
ActionController::RoutingError:
  No route matches [GET] "/images/branding/googlelogo/2x/googlelogo_color_92x30dp.png"```

I found this workaround to make Capybara not raise error to solve error above.
Do you have any ideas?